docs/research/wallet-comparison.md

```diff
@@ -0,0 +1,177 @@
+  - Similar to Breez it is unclear what happens when clicking on one of the app icons in the marketplace.
+- One-click wallet create for setup, but onboarding not as clear and rich as Phoenix
+
+Con:
+
+- Cannot "empty the wallet", always have to keep at least some sats in for "paying fees"
+  - This is super annoying, it is literally impossible to empty this wallet.
+- Multi-wallet management
+  - The wallets are completely independent; sending money from one to the other is quite cumbersome.
+  - I would have expected that I can easily move funds from one wallet to another, but this is quite cumbersome.
+  - Not sure what the use cases behind multiple wallets is; it does not feel like you average "bank account" app where you divide your money into different accounts for different purpose.
+- Send / Receive are not as intuitive as for Phoenix
+  - When using Phonex it becomes clear how Lightning works, even when one is new to send/receive and creating/paying invoices.
+  - The Blue wallet interface can be confusing, because it is not clear what one has to do to send/receive.
+- When using LndHub the backup is an lndhub link or a QR code. There is not 12-word seed available that can just be copied.
+  - How would I scan a QR code for backup with only one phone?
+  - Unclear what the "backup link" represents.
+
+For details check these links:
+
+- https://bluewallet.io/features/
+
+### Breez
+
+In a nutshell: Non-custodial setup similar to Phoenix and a good UX; includes features that go beyond just a wallet.
+
+I think Breez is currently the biggest competitor to what we have planned, because they already include "trading services".
+
+Setup:
+
+- Non-custodial setup similar to Phoenix but with more control options
+  - Run own LND node on Phone (forked version of LND + Neutrion Bitcoin node)
+    - Looks like they are the only one running both LND and Neutrino on the phone
+  - Channel management through Async nodes
+  - Can configure own Electrum Server
+  - Cannot configure own Lightning node (outside the phone setup)
+- Where is trust involved?
+  - Channel management
+  - Note: Backup is independent of Breez (different than Phoenix)
+
+Pro:
+
+- Fund wallet with BTC
+- Fund wallet with Fiat: Buy BTC feature enabled through MoonPay integration
+- Channel management
+  - Default
+  - Possibility to interact with Lightning node through basic interface for devs in Preferences
+
+Pro / Con:
+
+- Marketplace integrations
+  - e.g.: LN Markets: Can directly use funds to trade there
+  - App integrations are not native to the app, some work nice, some work less nice. The overall UX is not very good because it's mostly an integration of web-apps into breez.
+  - It would help to know what happens when clicking on the app icon
+    - This is more of an app-store experience similar to e.g. Umbrel; it might be better to have an overivew of what an app is and what happens when opening (installing?) it.
```

Comment:
```suggestion
    - This is more of an app-store experience similar to e.g. Umbrel; it might be better to have an overview of what an app is and what happens when opening (installing?) it.
```